leader-election-impl/src/main/java/com/palantir/paxos/SingleLeaderPinger.java

```diff
@@ -46,15 +49,17 @@
 import com.palantir.sls.versions.VersionComparator;
 
 public class SingleLeaderPinger implements LeaderPinger {
-
     private static final Logger log = LoggerFactory.getLogger(SingleLeaderPinger.class);
+    private static final RateLimiter pingV2RateLimiter = RateLimiter.create(1.0 / (5 * 60));
```

Comment:
Let's not use statics - this should likely be injected into each SingleLeaderPinger.

This is "once every 5 minutes" - could you include a comment to that effect, and/or extract a constant explaining what this value is?